packages/smooth_app/lib/pages/home_page.dart

```diff
@@ -272,7 +277,14 @@ class _HomePageState extends State<HomePage> {
   ) {
     final List<String> orderedVariables =
         userPreferencesModel.getOrderedVariables(userPreferences);
-    final List<String> attributes = <String>[];
+    final List<Widget> attributes = <Widget>[];
+    final Map<String, MaterialColor> colors = <String, MaterialColor>{
```

Comment:
I'm not sure if green / orange / red is a good choice to convey the importance. For other features we use green and red to convey if something is ok or not. Maybe we could change that to a single color, but with different intensity as it becomes more important. (same for how we display the titles of the sliders in the users preference).

But it's relatively unrelated to the rest of the PR, so we can merge it as is, and we can always discuss the colors later.

Reply:
At least they're in the correct order, and it still looks better in rounded rects than in plain text like before.
Colors and sizes can definitely be improved later.
As you know I'm not a big fan of the user preferences page anyway: I guess both could be refactored in the same style.

Just an idea: instead of the endless list of attributes with sliders, 4 pools of rounded rects: mandatory, very_important, ... And a drag and drop gesture to put the attributes in the desired pool.